src/classes/Loaders/EditModeLoader.ts

```diff
@@ -14,7 +14,7 @@ export class EditModeLoader implements ITimesheetLoader {
 
   constructor(pageTitle: string, timesheetTable: Element) {
     this.pageTitle = pageTitle;
-    this.todayDate = moment();
+    this.todayDate = moment().startOf("day");
```

Comment:
ℹ️ Did this throughout the PR too because why not? Ensures we're working with the start of a given date so it can't be thrown off by a few hours' difference.